packages/stylelint-config/index.ts

```diff
@@ -0,0 +1,67 @@
+import type { Config } from "stylelint";
+
+const config: Config = {
```

Comment:
We never mentionned if we wanted to use it or not. 

It could be possible with either a .prettierrrc file or via a hardcoded config in stylelint 
![image](https://user-images.githubusercontent.com/38871812/231865157-843f5ce7-a1de-4d95-a1ff-42cc6886c910.png)

Reply:
If we want it, i'll add it in another PR